src/client/hoc/withIdConnect.js

```diff
@@ -0,0 +1,21 @@
+import {connect} from 'react-redux';
+import {uploadIdMovieAction} from '@root/client/reduxStore/actions';
+
+export const withIdConnection = (Component) =>{
+
+    const mapStateToProps = (state) => {
```

Comment:
you could do next

`const mapStateToProps = (state) => ({
sLoading: state.movieIdState.isLoading,
movie: state.movieIdState.movie,
)}`

Reply:
Done)